tests/lib/version.py

```diff
@@ -0,0 +1,24 @@
+#
```

Comment:
Please name the file `version_test.py`, otherwise pytest does not discover it. It needs to have `test` in its filename.

Reply:
Oops. Done! Thanks :-)